test/fixtures.spec.ts

```diff
@@ -16,18 +16,42 @@
  */
 import { options } from './playwright.fixtures';
 import './remoteServer.fixture';
+import { registerFixture } from '@playwright/test-runner';
 
 import { execSync } from 'child_process';
 import path from 'path';
 
+declare global {
+  interface TestState {
+    wrapper: TestState['remoteServer'];
+    stallingWrapper: TestState['stallingRemoteServer'];
+  }
+}
+
+registerFixture('wrapper', async ({browserType, remoteServer, server}, test) => {
```

Comment:
This name is too generic, please use something that is clearly scoped to this test!

Reply:
Done.